src/app/window-handler.ts

```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
     }
 
+    /**
+     * HACK SWITCH to Client 1.5
+     */
+    private async onClient1_5(): Promise <void> {
+        logger.info('window handler: go to Client 1.5');
+        logger.info('this.globalConfig.url: ' + config.getGlobalConfigFields([ 'url']));
+
+        if (config.getGlobalConfigFields([ 'url']).url && config.getGlobalConfigFields([ 'url']).url.startsWith('https://corporate.symphony.com')) {
+            logger.info('window handler: switchClient: ' + switchClient);
+            if (switchClient === 'client1_5' ) {
+                return;
+            }
+
+            switchClient = 'client1_5';
+
+            const focusedWindow = BrowserWindow.getFocusedWindow();
+
+            const dogfoodUrl = `https://corporate.symphony.com/`;
+            if (focusedWindow && windowExists(focusedWindow)) {
+                await focusedWindow.loadURL(dogfoodUrl);
+                reloadWindow(focusedWindow as ICustomBrowserWindow);
+            } else {
+                logger.error('window handler: Could not go to client 1.5');
+            }
+        } else {
+            logger.info('Switch not support for this POD-url');
+        }
+    }
+
+    /**
+     * HACK SWITCH to Client Mana-stable
+     */
+    private async onClientManaStable(): Promise <void> {
+        logger.info('window handler: go to Client Mana-stable');
+
+        if (config.getGlobalConfigFields([ 'url']).url && config.getGlobalConfigFields([ 'url']).url.startsWith('https://corporate.symphony.com')) {
+            logger.info('window handler: switchClient: ' + switchClient);
+            if (switchClient === 'mana_stable') {
+                return;
+            }
+
+            switchClient = 'mana_stable';
+            const focusedWindow = BrowserWindow.getFocusedWindow();
+
+            if (focusedWindow) {
+                logger.info('window handler: windowExists: ' + windowExists(focusedWindow));
+            } else {
+                logger.info('window handler: no mainWindow');
+            }
+            let csrfToken;
+            if (focusedWindow && windowExists(focusedWindow)) {
+                try {
+                    csrfToken = await focusedWindow.webContents.executeJavaScript(`localStorage.getItem('x-km-csrf-token')`);
+                } catch (e) {
+                    logger.error(e);
+                }
+
+                const dogfoodUrl = `https://corporate.symphony.com/client-bff/index.html?x-km-csrf-token=${csrfToken}`;
+                await focusedWindow.loadURL(dogfoodUrl);
+                reloadWindow(focusedWindow as ICustomBrowserWindow);
+            } else {
+                logger.error('window handler: Could not go to client Mana-stable');
+            }
+        } else {
+            logger.info('Switch not support for this POD-url');
+        }
+    }
+
+    /**
+     * HACK SWITCH to Client Mana-daily
+     */
+    private async onClientManaDaily(): Promise <void> {
```

Comment:
We can completely remove this function and use the above function to load the URL dynamically